AWSome.Bucket/Config/BucketException.cs

```diff
@@ -1,10 +1,18 @@
 ﻿using Amazon.S3;
 using System;
 
-namespace AWSome.Bucket.Config
+namespace Liberator.AWSome.Bucket.Config
 {
+    /// <summary>
+    /// 
```

Comment:
Summary is empty?

Reply:
This exceptional bucket is no longer empty